### PR TITLE
Add okta push device support

### PR DESCRIPTION
**Pull request by badie**
- Allows us to use okta push
- Fixes null pointer encountered when attempting to select devices

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,7 +4,10 @@
 #
 # This makefile is meant for humans
 
-VERSION := $(shell git describe --tags --always --dirty="-dev" | cut -d- -f 1)
+## In concourse, before release, tag is of format 'vrc-1585761231-0.1.3-f6d7b72'
+## after release,it is of format 'v0.1.3'. Code below will work for both format
+## as cut will return the string if no delimiters are found
+VERSION := $(shell git describe --tags --always | cut -d- -f 3 | tr -d 'v')
```
Comment (switj) [inline Makefile:10]: Are we able to get this from an environment variable instead?

File: cmd/add.go [hunk 2]
```diff
@@ -7,9 +7,9 @@ import (
 	log "github.com/sirupsen/logrus"
 
 	"github.com/99designs/keyring"
-	analytics "github.com/segmentio/analytics-go"
 	"github.com/autonomic-ai/aws-okta/lib"
 	"github.com/autonomic-ai/aws-okta/lib/client"
+	analytics "github.com/segmentio/analytics-go"
```
Comment (switj) [inline cmd/add.go:12]: how about removing segmentio analytics?

File: lib/client/mfa/totp.go [hunk 3]
```diff
@@ -17,7 +17,7 @@ func (d *TOTPDevice) Supported(factor Config) error {
 	if factor.FactorType == "token:software:totp" || factor.FactorType == "token:hardware" {
 		return nil
 	}
-	return fmt.Errorf("sms doesn't support %s %w", factor.FactorType, types.ErrNotSupported)
+	return fmt.Errorf("totp doesn't support %s %w", factor.FactorType, types.ErrNotSupported)
```
Comment (switj) [inline lib/client/mfa/totp.go:20]: 🙏

File: Makefile (hunk 1 above)
Comment (badie) [reply Makefile:10]: Not possible anymore. versions are now controlled with git tags

File: cmd/add.go (hunk 2 above)
Comment (badie) [reply cmd/add.go:12]: I'll take it out as part of another PR

**Comment by badie** [comment]
@switj We can catch up after standup this morning

**Pull request merged**